### PR TITLE
Update resistor-color-trio files

### PR DESCRIPTION
**Pull request by BNAndras**
In syncing the tests, there's a test to ignore additional colors. Since I had to update example.lua for the additional unit conversions, I also changed decode to take an array. However, I'm fine with changing that part back if we want to mark that test as not included.  Although If we want to keep the test, I can do a variable number of arguments like `decode(c1, c2, c3, ...)` and not break existing student solutions. The solutions will still need to be updated for the additional conversions, but at least it'd be just that part that needs to be updated.

### CODE REVIEW

File: exercises/practice/resistor-color-trio/resistor-color-trio_spec.lua [hunk 1]
```diff
@@ -2,32 +2,63 @@ local rct = require 'resistor-color-trio'
...
+    assert.are.equal(8, value)
+    assert.are.equal('ohms', unit)
+  end)
+
+  it('ignore extra colors', function()
+    local value, unit = rct.decode({'blue', 'green', 'yellow', 'orange'})
+    assert.are.equal(650, value)
+    assert.are.equal('kiloohms', unit)
+  end)
+
```
Comment (ryanplusplus) [inline exercises/practice/resistor-color-trio/resistor-color-trio_spec.lua:63]:
```suggestion
```

**Comment by ryanplusplus** [comment]
Hey @BNAndras, thanks for putting this together! As far as taking an array, I'd prefer to keep taking individual named arguments for the sake of not breaking existing solutions. I don't think that a variable number of arguments is necessary as Lua ignores additional arguments and using 3 named arguments instead of `...` is a valid way to pass that test.

**Comment by BNAndras** [comment]
My bad. I confused the warning from the Lua language server about decode only expecting three arguments as an actual error. I'll fix momentarily

**Review by ryanplusplus — APPROVED** [review]

**Comment by ryanplusplus** [comment]
> My bad. I confused the warning from the Lua language server about decode only expecting three arguments as an actual error. I'll fix momentarily

No worries, thanks again!

**Pull request merged**